src/cc_catalog_airflow/dags/provider_api_scripts/europeana.py

```diff
@@ -0,0 +1,285 @@
+    prev_total = 0
+    for image_data in image_list:
+        total_images = _process_image_data(image_data)
+        if total_images is None:
+            total_images = prev_total
+        else:
+            prev_total = total_images
+
+    return total_images
+
+
+def _process_image_data(image_data):
+    logger.debug(f'Processing image data: {image_data}')
+    license_url = _get_license_url(image_data.get('rights'))
+    image_url = image_data.get('edmIsShownBy')[0]
+    foreign_landing_url = _get_foreign_landing_url(image_data)
+    foreign_id = image_data.get('id')
+    thumbnail_url = image_data.get('edmPreview')[0]
+    title = image_data.get('title')[0]
+    meta_data = _create_meta_data_dict(image_data)
+
+    return image_store.add_item(
+        foreign_landing_url=foreign_landing_url,
+        image_url=image_url,
+        license_url=license_url,
+        thumbnail_url=thumbnail_url,
+        foreign_identifier=foreign_id,
+        title=title,
+        meta_data=meta_data,
+    )
+
+
+def _get_license_url(license_field):
+    if len(license_field) > 1:
+        logger.warning('More than one license field found')
+    for license in license_field:
+        if 'creativecommons' in license:
+            return license
+    return None
+
+
+def _get_foreign_landing_url(image_data):
+    original_url = image_data.get('edmIsShownAt')
+    if original_url is not None:
+        return original_url[0]
+    europeana_url = image_data.get('guid')
+    return europeana_url
+
+
+def _create_meta_data_dict(
+        image_data
+):
+    meta_data = {
+        'country': image_data.get('country'),
+        'dataProvider': image_data.get('dataProvider'),
+        'description': _get_description(image_data)
+    }
+
+    return {k: v for k, v in meta_data.items() if v is not None}
+
+
+def _get_description(image_data):
+    if (
+            image_data.get('dcDescriptionLangAware') is not None
+            and image_data.get('dcDescriptionLangAware').get('en') is not None
+    ):
+        description = image_data.get('dcDescriptionLangAware').get('en')[0]
+    elif (
+            image_data.get('dcDescriptionLangAware') is not None
+            and image_data.get('dcDescriptionLangAware').get('def') is not None
+    ):
+        description = image_data.get('dcDescriptionLangAware').get('def')[0]
+    elif image_data.get('dcDescription') is not None:
+        description = image_data.get('dcDescription')[0]
+    else:
+        description = None
+
+    description = description.strip(
+    ) if description is not None else ''
+
+    return description
+
+
+def _build_query_param_dict(
+        start_timestamp,
+        end_timestamp,
+        cursor,
+        api_key=API_KEY,
+        default_query_param=DEFAULT_QUERY_PARAMS,
+):
+    query_param_dict = default_query_param.copy()
+    query_param_dict.update(
+        wskey=api_key,
+        query=f'timestamp_created:[{start_timestamp} TO {end_timestamp}]',
+        cursor=cursor,
+    )
+    
+
```

Comment:
Please remove the extra line, and make sure blank lines don't contain whitespace characters.
```suggestion

```